pages/event.tsx

```diff
@@ -0,0 +1,61 @@
+import { GetServerSideProps } from 'next'
+
+// components
+import Layout from '../components/Layout'
+import Navbar from '../components/Navbar'
+import Footer from '../components/Footer'
+import Container from '../components/Layout/Container'
+
+import picture from '../public/dummypost.png'
+
+// modules
+import HappeningNow from '../modules/EventPage/HappeningNow'
+import UpcomingSection from '../modules/EventPage/UpcomingSection'
+
+export interface PostProps {
+  userId: number
+  id: number
+  title: string
+  body: string
+}
+
+export interface PostsProps {
+  posts: PostProps[]
+}
+
+export interface SectionProps {
+  posts: PostProps[]
+  picture: StaticImageData
+}
```

Comment:
move `SectionProps` to relevant file,

Reply:
Is it okay to duplicate this interface in Both event modules? As this interface uses in two different places in Event modules